ideascube/mediacenter/templates/mediacenter/index.html

```diff
@@ -34,5 +34,5 @@
     {% include 'search/order_by.html' %}
     {% include 'search/filter_by_kind.html' %}
     {% include 'search/filter_by_lang.html' %}
-    {% tag_cloud url='mediacenter:index' model=view.model tags=available_tags %}
+    {% tag_cloud url='mediacenter:index' model=view.model limit=40 tags=available_tags %}
```

Comment:
I'm not sure that's really going to mitigate the issue all that much. :-/

```
$ for DB in *.sqlite; do sqlite3 $DB 'select count(id) from taggit_tag'; done
132
119
160
452
348
124
124
172
119
125
```

Granted, most of these tags are wrong (duplicates like `Sylvain` and `Sylvain;`, punctuation-only tags like `.`, etc.). But if we don't display them, library staff won't even know those bad tags exist and need to be fixed.

I'm not sure we can truly fix the issue without thinking hard about the form UI (auto-completion, better display of tags as they are parsed, ...), but simply the limit to 40 seems like it wouldn't achieve much at all. :-/

Maybe we should bump to 100? Maybe we should order the tags by most "frecent"? (i.e some heuristics based on "frequency" and "recency" of use)

(removing the limit entirely and displaying the 452 tags would make Ideascube look terrible, and we probably shouldn't do it)